primitives/core/src/sr25519.rs

```diff
@@ -547,11 +547,14 @@ pub mod vrf {
 	#[cfg(feature = "full_crypto")]
 	use crate::crypto::VrfSigner;
 	use crate::crypto::{VrfCrypto, VrfVerifier};
-	pub use schnorrkel::vrf::VRF_OUTPUT_LENGTH;
-	use schnorrkel::{errors::MultiSignatureStage, vrf::VRF_PROOF_LENGTH, SignatureError};
+	use schnorrkel::{
+		errors::MultiSignatureStage,
```

Comment:
we're going to deprecate the current multi-sig there btw, but maybe that does not help you right now

Reply:
I know that `schnorrkel 0.11` is coming soon. As soon as it is released there will be a follow up PR with renaming of `VrfOutput -> VrfPreOut`, and all not required stuff removal.

This PR is mostly to prepare the soil for a painless introduction of bandersnatch-vrf